src/main/java/org/spburegistry/backend/controller/TagController.java

```diff
@@ -23,6 +23,11 @@ public Iterable<TagTO> getAllTags() {
         return tagService.findAll();
     }
 
+    @GetMapping("/sortedByWeight")
+    public Iterable<TagTO> getAllTagsSortedByWeight() {
+        return tagService.findAllTagsSortedByWeight();
+    }
```

Comment:
I think it should be implemented in 1 complex request like '/tags?sortedByWeight=true&filterSubstring="sbstr"'

Reply:
Done